data/src/main/java/com/linkedin/data/schema/resolver/FileDataSchemaResolver.java

```diff
@@ -241,7 +266,7 @@ protected DataSchemaLocation transform(String path)
           StringBuilder builder = new StringBuilder();
           // within a JAR file, files are treated as resources. Thus, we should lookup using the resource separator
           // character, which is '/'
-          builder.append(InternalConstants.PEGASUS_DIR_IN_JAR)
+          builder.append(_schemasDirPathSuffix)
```

Comment:
if it is the first part of the string, the variable `_schemasDirPathSuffix` seems to be suitable by calling pathPrefix?

Reply:
please refer other comment also. its file/ schema name prefix not path

Reply:
removed suffix from name